backend/src/models/highscore.dto.ts

```diff
@@ -6,18 +6,22 @@ export class HighscoreDto {
   botName: string;
   @ApiModelProperty()
   score: number;
+  @ApiModelProperty()
+  season: string;
```

Comment:
Can we link to the id of the season instead?

Reply:
Fixing this.